packages/dx-react-grid-bootstrap3/src/templates/table-container.jsx

```diff
@@ -16,6 +16,8 @@ export const TableContainer = ({
       WebkitOverflowScrolling: 'touch',
       border: 0,
       margin: 0,
+      // NOTE: fix sticky positioning in Safari
```

Comment:
You can test these styles and delete this comment

Reply:
https://github.com/DevExpress/devextreme-reactive/blob/master/packages/dx-react-grid-material-ui/src/templates/table-container.jsx#L11